packages/sdk/src/Incentives/ERC20PeggedIncentive.ts

```diff
@@ -231,6 +232,22 @@ export class ERC20PeggedIncentive extends DeployableTarget<
     });
   }
 
+  /**
+   * The address of the pegged ERC20-like token
+   *
+   * @public
+   * @async
+   * @param {?ReadParams} [params]
+   * @returns {Promise<Address>}
+   */
+  public async peg(params?: ReadParams) {
```

Comment:
[I think this covers the need already](https://github.com/boostxyz/boost-protocol/blob/57f089f706a0dc52b3e26de3177068c3a05b20f1/packages/sdk/src/Incentives/ERC20PeggedIncentive.ts#L366)